assets/aircraft/atp.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "British Aerospace ATP",
+    "icao": "ATP",
+    "engines": {
+        "number": 2,
+        "type": "S"
+    },
+    "weightClass": "M",
+    "category": {
+        "srs": 3,
+        "lahso": 6,
+        "recat": "D"
+    },
+    "ceiling": 25000,
+    "rate": {
+        "climb":      1000,
+        "descent":    1500,
+        "accelerate": 3,
```

Comment:
Yeah, I have no idea what to use for accelerate and decelerate. These are the values from the Jetstream 32 I did, which in turn are the same as the Jetstream 31.

Reply:
It's an estimate, I'd say 3 is fine :)